### PR TITLE
Feat/shell bootloader baudrate

### PR DESCRIPTION
**Pull request by Salem-Tho**

### CODE REVIEW

File: pyluos/tools/discover.py [hunk 1]
```diff
@@ -4,18 +4,19 @@
...
-def serial_discover():
+def serial_discover(baudrate=1000000):
     serial_hosts = serial_ports()
     available_serial = []
     print("Searching for a gate available")
     for serial_host in serial_hosts:
         print("Testing " + str(serial_host))
         try:
-            port = serial.Serial(serial_host, os.getenv('LUOS_BAUDRATE', 1000000), timeout=0.05)
+            port = serial.Serial(serial_host, baudrate, timeout=0.05)
```
Comment (JeromeGalan) [inline pyluos/tools/discover.py:19]: Why does 'os.getenv' has disapeared ?

Comment (JeromeGalan) [reply pyluos/tools/discover.py:19]: It's OK, baudrate parameter is passed in the main function with getenv.

**Review by JeromeGalan — APPROVED** [review]

File: pyluos/tools/bootloader.py [hunk 2]
```diff
@@ -435,14 +435,15 @@ def reboot_network(device, nodes_to_reboot):
 # *******************************************************************************
 def luos_flash(args):
     print('Luos flash subcommand with parameters :')
+    print('\tbaudrate : ', args.baudrate)
```
Comment (nicolas-rabault) [inline pyluos/tools/bootloader.py]: this should be `--baudrate` to have something similar to the other ones.

**Review by nicolas-rabault — CHANGES_REQUESTED** [review]
Only one small thing.
@Salem-Tho you are officially a python dev ;)

**Review by nicolas-rabault — APPROVED** [review]
Nice

**Pull request merged**